server/athenian/api/openapi/openapi.yaml

```diff
@@ -310,9 +344,51 @@ paths:
 # {% endif %}
       x-openapi-router-controller: athenian.api.controllers.invitation_controller
       x-codegen-request-body-name: body
-  /metrics_line:
+  /metrics/code_bypassing_prs:
+    post:
+      operationId: calc_code_bypassing_prs
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CommitFilter'
+        description: Query for measuring the amount of code that was pushed outside
+          of pull requests.
+        required: true
+        x-body-name: body
+      responses:
+        200:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MeasuredCodeBypassingPRs'
+          description: Calculated metrics.
+        400:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/InvalidRequestError'
+          description: The request was invalid.
+        403:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GenericError'
+          description: The user is forbidden to access the specified repositories
+            on behalf of the specified account.
+      security:
+      - bearerAuth: []
+      summary: Measure the amount of code that was pushed outside of pull requests.
+      tags:
+        - metrics
+        # {% if False %}
+        - default
+      # {% endif %}
+      x-openapi-router-controller: athenian.api.controllers.metrics_controller
+      x-codegen-request-body-name: body
+  /metrics/pr_linear:
```

Comment:
What's the meaning of this new name? Why not `/metrics/prs`?

Reply:
Well, the original idea was to provide the "linear" metric values. At this stage, it has mutated into something PR-specific, so yes, we can make it `/metrics/prs`.